src/main.zig

```diff
@@ -6835,13 +6838,17 @@ const ClangSearchSanitizer = struct {
                 m.iframeworkwithsysroot = true;
                 if (m.iwithsysroot) warn(wtxt, .{ dir, "iframeworkwithsysroot", "iwithsysroot" });
             },
+            .embed_dir => {
+                if (m.embed_dir) return;
+                m.embed_dir = true;
+            },
         }
         try argv.ensureUnusedCapacity(ally, 2);
         argv.appendAssumeCapacity(arg);
         if (!joined) argv.appendAssumeCapacity(dir);
     }
 
-    const Group = enum { I, isystem, iwithsysroot, idirafter, iframework, iframeworkwithsysroot };
+    const Group = enum { I, isystem, iwithsysroot, idirafter, iframework, iframeworkwithsysroot, embed_dir };
```

Comment:
1. I'm not sure if `embed_dir` should be in this. What does it conflict with? I *assume* you can have header files and embed files in the same directory.
2. If it for some reason did need to warn about it, there's no warning implemented. This isn't doing anything.

Reply:
Wouldn't it just ignore the second arugment silently?

Reply:
That makes sense, since it's probably not intentional to specify the same directory twice (especially when ordering does matter, e.g. when two files in two search directories have the same name), although all of the other types do silently ignore duplicates for some reason.

Reply:
I think I'm going to leave it as-is for now, since I imagine there is/was a purpose to other types of search paths being silently ignored, and changing that behavior would be beyond the scope of this PR.